docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx

```diff
@@ -1,5 +1,5 @@
 Component to render UI part of enclosed MessageInput component.
-Please read section in guides about [Custimizing Message Input](/reactnative/guides/message-input).
+Please read section in guides about [Customizing Message Input](/reactnative/guides/message-input-customization).
```

Comment:
Do we need to include this change in the v4.0.0 docs as well?

Reply:
Umm, depends we can keep it against develop or v4. What's your call?

Reply:
Already fixed for the `v4` version in the past.

Reply:
Woops, my bad :D